src/interface.rs

```diff
@@ -86,6 +85,14 @@ impl Interface {
                     self.camera_controller.process_scroll(delta);
                     true
                 }
+                WindowEvent::MouseInput {
+                    button: winit::event::MouseButton::Left,
+                    state,
+                    ..
+                } => {
+                    self.camera_controller.mouse_captured = state == &ElementState::Pressed;
```

Comment:
We should probably only capture, and then have the ESC key handle releasing the mouse. This will only capture the mouse when the button is pressed. when you release the button, it will no longer be captured.

Look at the code in `CameraController::process_keyboard` for the ESC key. try and emulate that here, but only setting mouse captured to true. let the escape key handle releasing the mouse.

We also want to hide the cursor.

Reply:
> Look at the code in CameraController::process_keyboard for the ESC key. try and emulate that here, but only setting mouse captured to true. let the escape key handle releasing the mouse.
Ah, I see. :+1: 

> This fix will only capture the mouse when the button is pressed. when you release the button, it will no longer be captured.
Maybe I misunderstanding something, but I tought that's the goal? :)

Reply:
I guess its up to us on how we want to implement this. I prefer the traditional FPS control to the click and drag, but I am open to anything. maybe make it configurable in the ui?

Reply:
I didn't realize that ESC grabs the mouse. That's why I tought that the event handle was removed by mistake. Idk what is the traditional FPS control, but I'm perfrectly fine with pressing ESC. So you can close this :)

Reply:
If you want, you can add the cursor grab and hide to the first left mouse click here (only grab and hide, do not release) and let escape handle releasing the mouse.

Reply:
No, I just got used to how 3d tools (like blender/godot/pangolin...) work so I instinctively went that way. But FPS works for me too :+1: